### PR TITLE
Resolve conflicts on dev branch

### PR DESCRIPTION
**Pull request by CBroz1**

### CODE REVIEW

File: element_lab/lab.py [hunk 1]
```diff
@@ -48,24 +59,46 @@ class Organization(dj.Part):
...
 
 
 @schema
 class User(dj.Lookup):
+    """Table for storing user information.
+
+    Attributes:
+        user ( varchar(32) ): User name.
+        user_email ( varchar(128) ): User email address.
+        user_cellphone ( varchar(32) ): User cellphone number.
```
Comment (JaerongA) [inline element_lab/lab.py:99]: `user_fullname` missing

File: element_lab/lab.py [hunk 2]
```diff
@@ -48,24 +59,46 @@ class Organization(dj.Part):
...
     location_description=''    : varchar(255)
     """
 
 
 @schema
 class UserRole(dj.Lookup):
+    """Roles assigned to a user or a job title.
+
+    Attributes:
+        user_role ( varchar(16) ): Role within the lab (e.g., PI, Postdoc, etc.).
```
Comment (JaerongA) [inline element_lab/lab.py:84]: (  varchar(24) ):

File: element_lab/lab.py [hunk 3]
```diff
@@ -87,13 +128,27 @@ class LabMembership(dj.Lookup):
...
     protocol_type           : varchar(32)
     """
 
 
 @schema
 class Protocol(dj.Lookup):
+    """Protocol specifics (e.g., protocol number and title).
+
+    Attributes:
+        protocol ( varchar(16) ): Protocol identifier.
```
Comment (JaerongA) [inline element_lab/lab.py:147]: `( varchar(36) )`

**Comment by CBroz1** [comment]
Thanks for your review @JaerongA! I've incorporated these changes

In the future, you can directly suggest small changes like these, which will give me a button to directly add and cite you as an author of the work. Formatting shown on item 6 [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)

**Pull request merged**